app/utility/base_object.py

```diff
@@ -103,3 +101,18 @@ def load(cls, dict_obj):
             return cls.schema.load(dict_obj)
         else:
             raise NotImplementedError
+
+
+class AppConfigGlobalVariableIdentifier:
+    @classmethod
+    def iter_app_config_variables(cls):
+        for config_key in BaseWorld.get_config().keys():
+            if config_key.startswith('app.'):
+                yield '#{%s}' % config_key
+
+    @classmethod
+    def is_global_variable(cls, variable):
+        for config_variable in cls.iter_app_config_variables():
+            if variable == config_variable:
+                return True
```

Comment:
Fixme: `is_global_variable()` receives a variable without the `#{}` wrapper, so this is broken at the moment.

Reply:
I can also put a check at the top of the function to see if `variable` even starts with `app.` and return False if not.